sdk/aqueduct/api_client.py

```diff
@@ -274,7 +275,16 @@ def get_workflow(self, flow_id: str) -> GetWorkflowResponse:
         url = self._construct_full_url(self.GET_WORKFLOW_ROUTE_TEMPLATE % flow_id, self.use_https)
         resp = requests.get(url, headers=headers)
         utils.raise_errors(resp)
-        return GetWorkflowResponse(**resp.json())
+
+        workflow_response = GetWorkflowResponse(**resp.json())
+        for work_flow_dag in list(workflow_response.workflow_dags.values()):
+            for operator in list(work_flow_dag.operators.values()):
```

Comment:
typo: `work_flow`

Also, for better abstraction, the API client should work at a lower layer. It shouldn't know anything about our dag operators, it should only understand backend response objects. I would recommend exposing a separate `export` function in the APIClient and moving the dag updating logic up into the caller.